modules/core/test/v2/unit/coins/sol.ts

```diff
@@ -39,7 +96,15 @@ describe('SOL:', function () {
   });
 
   it('should verify transactions', async function () {
-    await basecoin.verifyTransaction('placeholder').should.be.rejectedWith('verifyTransaction method not implemented');
+    const txParams = newTxParams();
+    const txPrebuild = newTxPrebuild();
+    const validTransaction = await basecoin.verifyTransaction({ txParams, txPrebuild, memo, feePayer, blockhash, durebleNonce });
+    validTransaction.should.equal(true);
+  });
+  it('shouldnt verify transactions', async function () {
+    const txParams = newTxParamsWhitError();
```

Comment:
Add failure cases one for recipients,  one for total amount, one durable nonce, one fee payer and one blockhash. 5 extra failure test cases.